ArduPlane/quadplane.cpp

```diff
@@ -1250,8 +1250,8 @@ void QuadPlane::get_pilot_desired_lean_angles(float &roll_out_cd, float &pitch_o
         pitch_out_cd *= ratio;
     }
 
-    // apply lateral tilt to euler roll conversion
-    roll_out_cd = 100 * degrees(atanf(cosf(radians(pitch_out_cd*0.01))*tanf(radians(roll_out_cd*0.01))));
+    // apply lateral tilt to euler roll conversion   
+    roll_out_cd = 100 * degrees(atanf(cosf(radians(pitch_out_cd*0.01))*tanf(radians(roll_out_cd*0.01))));//不知道他在说啥
```

Comment:
Translation: I don't know what he's talking about

Not really appropriate comment for the code.